### DISCUSSION TITLE
Default release type of @auto-it/conventional-commits

### DISCUSSION THREAD

**Discussion by hasparus** [intuit/auto#2003] [General]
I'm not creating an issue, because I may well be just confused here, but shouldn't the [`defaultReleaseType`](https://github.com/intuit/auto/blob/main/plugins/conventional-commits/src/index.ts#L110) of _@auto-it/conventional-commits_ be something like "none" or "internal"?

`skip-release` cancels whole release, even if other merged PRs since the last release are labeled as `minor`. (https://github.com/intuit/auto/blob/main/packages/core/src/semver.ts#L221-L227)

**Comment by hipstersmoothie** [comment]
I think to fit with the default auto experience yeah it would make sense to change that. Could you submit a PR?

**Reply by hasparus** [reply]
> Could you submit a PR?

Okay, I gave the rest of the code a brief look. Here are my observations.

The test at [conventional-commits.test.ts#L370-L401](https://github.com/intuit/auto/blob/main/plugins/conventional-commits/__tests__/conventional-commits.test.ts#L370-L401) actually guards against a similar scenario, but it doesn't catch the case where the user merges _one PR with chores only, which is then labeled as __skip-release___ and then a PR with a fix or feature. Since labels from merged PRs since the last release are merged, we end up skipping a release of these fixes.

I fixed the problem for myself by configuring `defaultReleaseType` to `"none"`, but maybe the problem lies in the logic which decides if the release is skipped based on _skip-release_ labels?

Right now, one _skip-release_ label is enough to skip the whole thing. Shouldn't Auto skip only if all PRs in the release contain the label?
